tasks/3_is-boolean/is-boolean.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * convertToF - convert from Celsius to Fahrenheit
+ *
+ * @param {*} bool
+ * @return {Boolean}
+ */
+export function isBoolean(bool) {
+  return typeof(bool) === 'boolean';
```

Comment:
typeof is an operator - not a function. Parentheses is optional here

Reply:
Fixed it

Reply:
Btw it's also allowed
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#Syntax